.github/workflows/helm-checks.yaml

```diff
@@ -0,0 +1,148 @@
+          echo "github.event.pull_request.head.repo.full_name == github.repository = ${{ github.event.pull_request.head.repo.full_name == github.repository }}"
+          echo "NON_FORKED_AND_NON_ROBOT_RUN = $NON_FORKED_AND_NON_ROBOT_RUN"
+          echo "NON_FORKED_AND_NON_ROBOT_RUN=$NON_FORKED_AND_NON_ROBOT_RUN" >> "$GITHUB_OUTPUT"
+
+  build_image:
+    name: Build Docker Image
+    needs: prepare_ci_run
+    runs-on: ubuntu-22.04
+    env:
+      BRANCH: ${{ needs.prepare_ci_run.outputs.BRANCH }}
+      DATETIME: ${{ needs.prepare_ci_run.outputs.DATETIME }}
+      BUILD_TIME: ${{ needs.prepare_ci_run.outputs.BUILD_TIME }}
+      GIT_SHA: ${{ needs.prepare_ci_run.outputs.GIT_SHA }}
+      RELEASE_REGISTRY: "localhost:5000/keptn"
+    strategy:
+      matrix:
+        config:
+          - name: "keptn-lifecycle-operator"
+            folder: "operator/"
+          - name: "scheduler"
+            folder: "scheduler/"
+          - name: "functions-runtime"
+            folder: "functions-runtime/"
+          - name: "klt-cert-manager"
+            folder: "klt-cert-manager/"
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Cache build tools
+        id: cache-build-tools
+        uses: actions/cache@v3
+        with:
+          path: ./${{ matrix.config.folder }}bin
+          key: build-tools-${{ github.ref_name }}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build Docker Image
+        uses: docker/build-push-action@v3
+        with:
+          context: ${{ matrix.config.folder }}
+          platforms: linux/amd64
+          target: production
+          tags: |
+            ${{ env.RELEASE_REGISTRY }}/${{ matrix.config.name }}:dev-${{ env.DATETIME }}
+          build-args: |
+            GIT_HASH=${{ env.GIT_SHA }}
+            RELEASE_VERSION=dev-${{ env.DATETIME }}
+            BUILD_TIME=${{ env.BUILD_TIME }}
+            CONTROLLER_TOOLS_VERSION=${{ env.CONTROLLER_TOOLS_VERSION }}
+            SCHEDULER_COMPATIBLE_K8S_VERSION=${{ env.SCHEDULER_COMPATIBLE_K8S_VERSION }}
+          builder: ${{ steps.buildx.outputs.name }}
+          push: false
+          cache-from: type=gha,scope=${{ github.ref_name }}-${{ matrix.config.name }}
+          cache-to: type=gha,scope=${{ github.ref_name }}-${{ matrix.config.name }}
+          outputs: type=docker,dest=/tmp/${{ matrix.config.name }}-image.tar
+
+      - name: Upload image as artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.config.name }}-image.tar
+          path: /tmp/${{ matrix.config.name }}-image.tar
+
+
+  build_helm_chart:
+    name: Build Helm Chart
+    needs: [ prepare_ci_run, build_image ]
+    runs-on: ubuntu-22.04
+    env:
+      RELEASE_REGISTRY: "localhost:5000/keptn"
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Generate helm charts
+        run: |
```

Comment:
Cannot we in the current state (without kustomize) generate them just with `make helm-package` ?